src/parser.cu

```diff
@@ -78,6 +78,7 @@ Grid parseArgs(int argc, char** argv){
                 int xDim = atoi(optarg);
                 printf("Argument for x is given as %d\n",xDim);
                 par.store("xDim",(int)xDim);
+                break;
```

Comment:
For some reason, this line was removed in my last PR. I have no idea how, but it managed to slip through.